src/libraries/Microsoft.Extensions.Logging.Abstractions/tests/Microsoft.Extensions.Logging.Generators.Tests/LoggerMessageGeneratorEmitterTests.cs

```diff
@@ -124,7 +139,6 @@ private async Task VerifyAgainstBaselineUsingFile(string filename, string testSo
                 new[] { typeof(ILogger).Assembly, typeof(LoggerMessageAttribute).Assembly },
                 new[] { testSourceCode }).ConfigureAwait(false);
 
-            Assert.Empty(d);
```

Comment:
Do we really want to shut this off for the other tests? Would it be better to pass a `bool` into this method to say whether to expect diagnostics or not?

Reply:
oh this was a mistake. I'll add this back